confmap/README.md

```diff
@@ -93,6 +93,91 @@ The `Resolve` method proceeds in the following steps:
 4. For each "Converter", call "Convert" for the "result".
 5. Return the "result", aka effective, configuration.
 
+#### (Experimental) Append merging strategy for lists
+
+You can opt-in to experimentally combine slices instead of discarding the existing ones by enabling the `confmap.enableMergeAppendOption` feature flag. Lists are appended in the order in which they appear in their configuration sources.
+This will **not** become the default in the future, we are still deciding how this should be configured and want your feedback on [this issue](https://github.com/open-telemetry/opentelemetry-collector/issues/8754).
```

Comment:
> This will **not** become the default in the future

Do we want to replace it with another feature gate in future? I think we can keep the feature gate and just keep changing its behavior. So, I would change the wording to something like "The behavior of this feature gate will change over time before it can be considered for beta stage".

However there have been a lot of discussion around that. So I maybe missing something

Reply:
SGTM 👍